### ISSUE TITLE
Support dimmed text

### ISSUE THREAD

**Issue by oir** [wader/ansisvg#52]
Hi, great project!

I noticed that dimmed text (ansi code 2) is not supported out of the box, would it be possible to support this?

If welcome, I can submit a PR for it. I have an implementation here that supports it by merely adding 0.5 opacity to the modified text (not familiar with Go at all so I mostly did pattern recognition 😅):
https://github.com/wader/ansisvg/compare/master...oir:ansisvg:dim#diff-5948cb3358520edabb7269cc310a861b2e2223a57e38e697aac25aeee34fd856

Example output [here](https://github.com/user-attachments/assets/b2edd125-956e-44f6-a12c-92b4569417da), see, e.g., the last row.

**Comment by wader** [comment]
Hey, yeap that seem resonable to support. But i do wonder how most terminal treats it, i think in some cases gets rendered as bold?

Maybe we could try `echo -e 'hello \e[2mdim \e[1mintense\e[0m world'
hello dim intense world` some terminals? this is iTerm2:

<img width="380" alt="Screenshot 2024-12-14 at 19 47 28" src="https://github.com/user-attachments/assets/b16fe5a3-5968-425c-81c2-4ee6ffb56235" />

Also do you know any popular programs that uses it that could be used as test cases?

No worries about being familiar with go, i try to keep the ansisvg code very basic and i i'm happy to help out if you create a PR.

@patrislav1 any thoughts? i have some vague memory that we did use css opacity for something at some point? 🤔

**Comment by oir** [comment]
One example (from python-land) that I know that uses it in the output is [uv](https://github.com/astral-sh/uv), e.g.:

```bash
❯ uv --color=always pip install requests &> tmp.txt
❯ cat -e tmp.txt
^[[2mUsing Python 3.11.10 environment at: env3.11^[[0m$
^[[2mAudited ^[[1m1 package^[[0m ^[[2min 2ms^[[0m^[[0m$
```

<img width="356" alt="image" src="https://github.com/user-attachments/assets/d4a27a77-96a1-4974-a9ec-62712c1f12e3" />

re: other terminals, I'm unsure. I have tested things only with iTerm2.

**Comment by wader** [comment]
@oir hey! added some tests to your changes and pushed a PR. Let's continue with details in https://github.com/wader/ansisvg/pull/53

**Comment by wader** [comment]
Closed by #53
